src/server.rs

```diff
@@ -137,7 +137,11 @@ impl Server {
         }
     }
 
-    fn update_clients(&mut self, delta_time: f32, hit_players: Vec<u64>) {
+    fn update_clients(
+        &mut self, delta_time: f32,
+        hit_players: Vec<u64>,
+        hit_powerup_positions: Vec<(u64, Point2<f32>)>
```

Comment:
Indent these lines.

Also, replace `Vec<_>` with `&[_]` since they are not modified